### PR TITLE
DOCS: add how to update table of content

### PR DESCRIPTION
**Pull request by tainguyenbui**
## Description
Add information on how to update table of contents for documentation

## Changes
- Add section to show how to update table of contents
- Update table of content script

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,6 +92,16 @@ Enhancement suggestions are tracked as GitHub issues. Make sure you provide the
 - Explain why this enhancement would be useful to most Booster users and isn't something that can or should be implemented as a community package.
 - List some other libraries or frameworks where this enhancement exists.
 
+### Improving documentation
+
+Booster documentation, located at `/docs/README.md`, is treated as a live document that continues improving on a daily basis. If you find something that is missing or can be improved, please contribute, it will be of great help for other developers.
+
+Bear in mind that if you have added a new section, or changed an existing one you will need to update the table of content. This can be easily done with the following command:
+
+```typescript
```
Comment (alvaroloes) [inline CONTRIBUTING.md:101]:
```suggestion
```sh
```

**Comment by tainguyenbui** [comment]
bot: integration

**Comment by alvaroloes** [comment]
@tainguyenbui Just wondering: There are only changes in .MD and .json files. Is there a specific reason why you run integration tests?

**Comment by tainguyenbui** [comment]
@alvaroloes I thought the verdions that pass integration tests get deployed automatically. Also, I wanted to check stability of the integration tests after the latest fixes

**Pull request merged**